test/tss2-tcti-tabrmd_unit.c

```diff
@@ -839,6 +847,8 @@ tcti_tabrmd_receive_poll_fail_test (void **state)
     uint8_t command_out [sizeof (command_in)] = { 0 };
     TSS2_RC rc;
 
+    (void)command_in;
```

Comment:
Assuming this is to cope with an unused parameter. Any reason you didn't use the `UNUSED_PARAM` macro from `src/util.h`?

Reply:
It's not a parameter.  But I will use it.

Reply:
Sounds like the macro is misnamed then: should just be `UNUSED`.

Reply:
I added a UNUSED_VAR macro.